features/support/commonlib.rb

```diff
@@ -34,3 +34,31 @@ def filename
 def current_url
   driver.current_url
 end
+
+# generate answers file to interactively run configure-proxy.sh.
+def generate_answers_file(server, proxy)
+  raise(ArgumentError, 'Hostname is missing!') if server.nil? || server.empty? || proxy.nil? || proxy.empty?
+  path = ''
+  Tempfile.open('config-answers.txt') do |f|
+    path = f.path
+    f.write("RHN_PARENT=#{server}\n")
+    f.write("HTTP_PROXY=''\n")
+    f.write("VERSION=3.2\n")
```

Comment:
Nitpick: remember to dont hardcode 3.2 Version number, since we want to chose servers

Reply:
This is "head" branch, I was planning to change this to 3.1 on "manager31" branch.

So there's not a big problem to hardcode if we use a different version on each git branch.

Reply:
The problem with this git branch isthat imagine that HEAD -> 3.3 . 
I think we should Pick up this variable from a file that susemanager offer or via spacecmd XMLRPC api if there is not a file that reveals the suse-manager version

Reply:
/usr/share/rhn/config-defaults # vim rhn_web.conf 
or 
using rpm -qa susemanager

Reply:
We can change this to 3.3 when head moves to 3.3, no big deal.

BTW in sumaform this value is still.... 2.1 !!! Apparently, this has no big consequences. I even wonder if it could be removed from the product.

Reply:
Ok thx ebischoff for the investigation. I think it is worth to ask if we can remove it.
If we have the possibilty i would not add a manual change to versions, this is simply unsafe, imagine  we start to rely on all this small, changes